albyhub/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: albyhub
+name: Alby Hub
+tagline: Your Hub to connect to the Bitcoin world
+category: finance
```

Comment:
I just noticed that the nostr wallet connect app is also listed in the `finance` category.

This means that both of the alby apps end up here in the app store:
<img width="1340" alt="image" src="https://github.com/user-attachments/assets/83898433-1501-4485-8d3b-a31735359010">

Instead of in the `bitcoin` category with related apps:
<img width="1340" alt="image" src="https://github.com/user-attachments/assets/10ff2728-0095-48cc-b960-416b078a4038">

It's totally up to you where you want this to render. If you'd like to switch to the bitcoin category then you can do:
```suggestion
category: bitcoin
```